app/models/edition.rb

```diff
@@ -36,6 +36,7 @@ class Edition < ApplicationRecord
   has_many :edition_authors, dependent: :destroy
   has_many :authors, through: :edition_authors, source: :user
   has_many :classification_featurings, inverse_of: :edition
+  has_many :link_check_reports, as: :link_reportable, class_name: LinkCheckerApiReport
   has_many :links_reports, as: :link_reportable
```

Comment:
Can we avoid 9bdd5e8 by just getting rid of the removal change in the earlier commit?